src/rules/__tests__/expect-expect.test.ts

```diff
@@ -28,67 +28,33 @@ ruleTester.run('expect-expect', rule, {
       options: [{ assertFunctionNames: ['expectSaga'] }],
```

Comment:
Could you add a test for what happens if you have `*` & `**` at the end? i.e `request.*` & `request.**`.

Also having ones for them standalone might be useful too, just to display the behaviour (`*` & `**`).

(It might be worth creating a new run specifically for the wildcard matching behaviour; something like `ruleTester.run('wildcards', rule, { ... });`)

Reply:
> Also having ones for them standalone might be useful too, just to display the behaviour (* & **).
Forgot this one. Making some additional changes right now

Reply:
Added the additional rules and also made sure that wildcard always match up until the next `.` or the end of the string